src/test/resources/oauth-authorize/init-data.sql

```diff
@@ -0,0 +1,21 @@
+DELETE FROM AUTHORIZATION_CODES;
```

Comment:
Bunun yerine repo pattern ile testlerden önce db hazırlayalım teste özel. dblerin durumu için aşağıdaki state managementi kullanalımö:

https://www.baeldung.com/spring-tests#7-state-management